src/main/java/uk/gov/hmcts/reform/ccd/document/am/service/impl/DocumentManagementServiceImpl.java

```diff
@@ -84,17 +82,17 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(DocumentManagementServiceImpl.class);
 
-    private RestTemplate restTemplate;
+    private transient RestTemplate restTemplate;
 
-    private SecurityUtils securityUtils;
+    private transient SecurityUtils securityUtils;
 
     @Value("${documentStoreUrl}")
-    protected String documentURL;
+    protected transient String documentURL;
 
     @Value("${documentTTL}")
-    protected String documentTtl;
+    protected transient String documentTtl;
 
-    private CaseDataStoreService caseDataStoreService;
+    private transient CaseDataStoreService caseDataStoreService;
```

Comment:
This will cause code smell issues as explained above.

Reply:
Done.